.github/workflows/e2e-functional.yml

```diff
@@ -65,17 +64,16 @@ jobs:
             github.event.pull_request.labels && 
             contains(github.event.pull_request.labels.*.name, 'Run Desktop E2E Tests')
         ) ||
-          (
-              github.event_name == 'schedule' || github.event_name == 'workflow_call'
-          )
+        (
+          (inputs.tag != '')
+        )
       }}
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
     steps:
-
       - name: ci/checkout-repo
         uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
       - name: ci/setup-node
-        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
+        uses: actions/setup-node@v4
```

Comment:
Please follow the same convention by using fix commit hash and version in comment. I believe the main `mattermost` repo is using v4 already.

Reply:
sure Saturn.